neuralprophet/__main__.py

```diff
@@ -0,0 +1,16 @@
+"""
+Invokes neuralprophet when module is run as a script.
+"""
+import argparse
+
+from neuralprophet._version import __version__
+
+
+def parse_args(args=None):
+    parser = argparse.ArgumentParser(description="NeuralProphet")
+    parser.add_argument("-v", "--version", action="version", version="%(prog)s " + __version__)
```

Comment:
Use capital letter `-V` for version shorthand

```suggestion
    parser.add_argument("-V", "--version", action="version", version="%(prog)s " + __version__)
```